System.IO.FileSystem/File.cs

```diff
@@ -13,39 +14,25 @@ namespace System.IO
     /// </summary>
     public static class File
     {
-        #region Constants
-
-        private const int _defaultCopyBufferSize = 2048;
-
-        #endregion
-
-
-        #region Static Methods
+        private const int ChunkSize = 2048;
+        private static readonly byte[] EmptyBytes = new byte[0];
 
         /// <summary>
         /// Copies an existing file to a new file. Overwriting a file of the same name is not allowed.
         /// </summary>
         /// <param name="sourceFileName">The file to copy.</param>
         /// <param name="destFileName">The name of the destination file. This cannot be a directory or an existing file.</param>
         /// <exception cref="ArgumentException">sourceFileName or destFileName is null or empty</exception>
-        public static void Copy(
-            string sourceFileName,
-            string destFileName)
-        {
-            Copy(sourceFileName, destFileName, false);
-        }
+        public static void Copy(string sourceFileName, string destFileName) => Copy(sourceFileName, destFileName, overwrite: false);

```

Comment:
This wrapped idented parameters style it's on purpose and is to be maintained.
Makes it easier to read and to check for differences on code comparison and changes.